system/logaccess/Grafana/CurlClient/GrafanaCurlClient.cpp

```diff
@@ -729,12 +732,30 @@ GrafanaLogAccessCurlClient::GrafanaLogAccessCurlClient(IPropertyTree & logAccess
             m_grafanaPassword.set(logAccessPluginConfig.queryProp("connection/@password"));
     }
 
-    //this is very important, without this, we can't target the correct datasource
-    fetchDatasourceByName(m_targetDataSource.name.get());
+    try
+    {
+        //this is very important, without this, we can't target the correct datasource
+        fetchDatasourceByName(m_targetDataSource.name.get());
+    }
+    catch(IException * e)
+    {
+        StringBuffer description;
+        IERRLOG("%s: datasource exception: (%d) - %s", COMPONENT_NAME, e->errorCode(), e->errorMessage(description).str());
```

Comment:
Should this be an OERRLOG?  Same for line 756?

Reply:
Given the below definitions, yes, OERRLOG is correct.

 *    (ii) Messages for SysAdmins:                                                      *
 *        OERRLOG([LogMsgCode,] format,..)     - uses MCoperatorError                   *
 *                                                                                      *
 *    (iii) Messages for platform developers:                                           *
 *        IERRLOG([LogMsgCode,] format,..)     - uses MCdebugError